src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -131,15 +139,47 @@ struct TimeDependentMapOptions {
     std::optional<std::variant<KerrSchildFromBoyerLindquist>> initial_values{};
   };
 
-  using options = tmpl::list<InitialTime, ShapeMapOptions>;
+  struct TranslationMapVelocity {
+    static std::string name() { return "Velocity"; }
+    using type = std::array<double, 3>;
+    static constexpr Options::String help = {
+        "Velocity of the translation map."};
+    using group = TranslationMapOptions;
+  };
+
+  struct TranslationMapInnerRadius {
```

Comment:
Don't take the inner and outer radii as options. Just apply the transition in the outer shell of the sphere, so inner radius is the last number in `radial_partitions` and outer radius is the outer radius of the sphere. Add this translation map with the transition _only_ to the blocks that make up the outer shell. Apply a rigid translation (no inner/outer radii) to all other blocks. If there is no partition (just one shell) and the interior of the sphere is filled with a cube then either error out or apply a rigid translation (because the transition only works with spherical inner and outer radii).